src/Tribe/Promoter/Observer.php

```diff
@@ -41,39 +41,40 @@ public function hook() {
 		$this->registered_types();
 
 		// Listen for changes on RSVP as Gutenberg Uses the post_type API to update RSVP's
-		add_action( 'save_post_tribe_rsvp_tickets', [ $this, 'notify_rsvp_event' ], 10, 2 );
+		add_action( 'save_post_tribe_rsvp_tickets', [ $this, 'notify_ticket_event' ], 10, 1 );
 
 		// RSVP
 		add_action( 'tickets_rsvp_ticket_deleted', [ $this, 'notify_event_id' ], 10, 2 );
 		add_action( 'event_tickets_rsvp_tickets_generated', [ $this, 'notify_event_id' ], 10, 2 );
 
+		// Moved tickets
+		add_action( 'tribe_tickets_ticket_type_moved', [ $this, 'ticket_moved_type' ], 10, 4 );
+		add_action( 'tribe_tickets_ticket_moved', [ $this, 'ticket_moved'], 10, 6 );
+
 		// PayPal
 		add_action( 'tickets_tpp_ticket_deleted', [ $this, 'notify_event_id' ], 10, 2 );
 		add_action( 'event_tickets_tpp_tickets_generated', [ $this, 'notify_event_id' ], 10, 2 );
 
 		// All tickets
 		add_action( 'event_tickets_after_save_ticket', [ $this, 'notify' ], 10, 1 );
-
-		// Actions from REST
-		add_action( 'tribe_tickets_ticket_type_moved', [ $this, 'ticket_moved_type' ], 10, 4 );
 	}
 
 	/**
-	 * Notify to the parent Event when an attendee has changes via REST API.
+	 * Notify to the parent Event of the ticket
 	 *
 	 * @since 4.10.1.2
 	 *
-	 * @param $attendee_id
+	 * @param $ticket_id int The Ticket ID where to look for the Event.
 	 */
-	public function notify_rsvp_event( $attendee_id ) {
+	public function notify_ticket_event( $ticket_id ) {
```

Comment:
Since we are modifying the method name here we should create the old one as a deprecated method and point to this one. Also update both of their Docblocks to reflect the change.